src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
+
+      &--compact {
+        size: 1.625rem;
+        font-size: var(--sapFontSize);
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        width: auto;
```

Comment:
You should avoid width: auto since it has problems with IE11 and we are supporting IE11

Reply:
Changes done.